utils/setupJest.ts

```diff
@@ -185,18 +184,19 @@ export const registerScreenshotReporter = () => {
   beforeEach(() => screenshotPromise);
   afterAll(() => screenshotPromise);
 
-  (jasmine as any).getEnv().addReporter({
-    specDone: async (result: any) => {
-      if (result.status === "failed") {
-        try {
-          await renameExtraLogs(result.fullName, result.status);
-        } catch (e) {
-          // eslint-disable-next-line no-console
-          console.error(e);
-          // Lets only log the error, so tno want any side effect.
-        }
-      }
-    },
-  });
+  // bump jest to 29.
```

Comment:
What's the status of these commented lines - should they stay like that?

Reply:
like that or removed, :gun: